testomania/src/main/java/com/earth/testomania/home_screen/presentation/HomeScreen.kt

```diff
@@ -236,6 +255,77 @@ fun CardButton(
     }
 }
 
+@Composable
+fun NetworkStateManager(
+    color: Color,
+    networkConnectivityObserver: NetworkConnectivityObserver,
+    initialNetworkState: Boolean
+) {
+    val status by networkConnectivityObserver.observe()
+        .collectAsState(initial = initialNetworkState)
+
+    if (status == ConnectivityObserver.ConnectionState.Unavailable) {
+        Pulsating {
+            Column(
+                Modifier
+                    .fillMaxSize()
+                    .padding(
+                        start = 26.dp,
+                        top = 80.dp,
+                        end = 26.dp,
+                        bottom = 0.dp
+                    )
+            ) {
+                Box(
+                    Modifier
+                        .fillMaxWidth()
+                        .height(44.dp)
```

Comment:
I suggest using `wrapContentHeight()` instead of hardcoded height, cause it will be problem in case User increases font size from system settings